resources/ext/cli/delete-reports.erb

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+set -e
+
+pdb_service=<%= EZBake::Config[:user] %>
+pdb_db_name="$pdb_service"
+<% if EZBake::Config[:build_type] == "pe" %>
+pg_user=pe-postgres
+jar_resource=puppetlabs/puppetdb/sql/delete_reports.sql
+<% else %>
+pg_user=postgres
+jar_resource=sql/delete_reports.sql
+<% end %>
+pdb_jar=/opt/puppetlabs/server/apps/puppetdb/puppetdb.jar
+
+function printerr {
+  echo "$@" 1>&2
+}
+
+function usage {
+  cat <<USAGE
+Usage: puppetdb delete-reports [OPTIONS]
+    --service SERVICE     the puppetdb service name (default: $pdb_service)
+    --db-name NAME        the puppetdb database name (default: $pdb_db_name)
+    --pg-user USER        the postgres system user (defult: $pg_user)"
+    --resource RESOURCE   the jar resource to extract (default: $jar_resource)
+    --jar PATH            the full path to the puppetdb jar file"
+                            (default: $pdb_jar)"
```

Comment:
Perhaps we should offer a variable for custom paths to the `jar` or `unzip` binaries.  The `pe-java8-devel` package installs it in `/opt/puppetlabs/server/apps/java/lib/jvm/java/bin/jar`, which is likely never going to be in the default PATH.